lm_eval/tasks/__init__.py

```diff
@@ -2,6 +2,7 @@
 from pprint import pprint
 from typing import List, Union
 
+from promptsource.templates import DatasetTemplates
```

Comment:
Remove this line. `DatasetTemplates` is already imported on line 1. Thanks!

Reply:
My bad! fixed this, thanks :)